tool/bots.sh

```diff
@@ -124,13 +124,19 @@ elif [[ "$BOT" == "test_ddc" || "$BOT" == "test_dart2js" ]]; then
     fi
     echo "USE_WEBDEV_RELEASE = $USE_WEBDEV_RELEASE"
 
+    FILES="test/"
+    if [ "$ONLY_GOLDEN" = "true" ]; then
+        # Set the test files to only those containing golden test
+        FILES=$(grep -rl "matchesDevToolsGolden" test | grep "_test.dart$" | tr '\n' ' ')
```

Comment:
There is a chance that someone could add a test that uses matchesGolden instead of matchesDevToolsGolden even though all golden checks should go through matchesDevToolsGolden. So we can do one of two things:

1) add a check on this job to make sure that no test files use 'matchesGolden'
2) check for 'matchesDevToolsGolden' and 'matchesGolden' in this search function

Reply:
kk I've added a 2nd match for us to filter on 😄